src/utils/color-utils.js

```diff
@@ -60,6 +60,25 @@ const colorUtils = {
     const current = getCurrentTheme(Theme);
     return (current && current.properties.fontFamily) || 'inherit';
   },
+  getFadedColor(color) {
+    const percent = -0.15;
+    let f; let R; let B; let G;
```

Comment:
I guess you could also write `let f, R, B, G;`

Reply:
I think the linter wanted it like this, I can check

Reply:
Crazy linters